thonny/running.py

```diff
@@ -916,8 +974,42 @@ def _get_environment(self):
             env["THONNY_DEBUG"] = "1"
         elif "THONNY_DEBUG" in env:
             del env["THONNY_DEBUG"]
+
+        add_env_file = self._load_env_file()
+
+        chained_env = collections.ChainMap(add_env_file, env)
+        chained_env = dict(chained_env)
+
+        # un-set keys
+        env = EnvFile()
+        env.set_env(chained_env)
+        env.remove_unused()
+
+        return env.get_env()
+
+    def _load_env_file(self):
+        # todo: configuration menu -> Tools > Options > Run & Debug
+        env_file = os.environ.get("THONNY_ENV_FILE", ".env")
```

Comment:
THONNY_ENV_FILE must be simply filename without path, right?

Reply:
no. 
it also can be a absolute, or relative path.

refer to [this](https://github.com/thonny/thonny/blob/6e7b9cf572a46cd95bdd38a5411064b9649d8b7f/thonny/running.py#L881) where it is just used with open.

Reply:
oops. small correction. i just went too quickly thru my own coding...
its used with path join - so it is just the filename without path.

Reply:
probably ".env" should be named as "thonny.env" instead;
since a dot-file is a hidden file under some systems e.g. raspberry pi.
just to apply to thonny's idea that it should be easy to understandable by beginners.
and also because hidden files are not shown by default.
but you decide.

Reply:
I'd go for `.env` as I don't think dotenv is begginner's topic.

Reply:
Ok